plugin.js

```diff
@@ -53,7 +53,7 @@ function plugin (fastify, options, next) {
   fastify.decorate('parseCookie', function parseCookie (cookieHeader) {
     return cookie.parse(cookieHeader, options.parseOptions)
   })
-  fastify.decorateRequest('cookies', {})
+  fastify.decorateRequest('cookies', () => ({}))
```

Comment:
Yes... that seems to work given that the tests all pass. However, look above at line 41:
https://github.com/fastify/fastify-cookie/blob/2c01183a9603c43bc252347bffa3642742496216/plugin.js#L41

The point of setting it to `{}` here is to establish the shape of the property. I'm not certain, but I don't think switching from a function to an object will prevent a de-optimization within V8. @mcollina do you know?

Reply:
Hum... maybe use `null` instead ?
The documentation states:
> Initialize a decorator as a '' if the intended value is a string, and as null if it will be an object or a function.